app/javascript/widget/api/contacts.js

```diff
@@ -12,9 +12,14 @@ export default {
       ...userObject,
     });
   },
-  setCustomAttibutes(customAttributes = {}) {
+  setCustomAttributes(customAttributes = {}) {
     return API.patch(buildUrl('widget/contact'), {
       custom_attributes: customAttributes,
     });
   },
+  deleteCustomAttributes(customAttributes = {}) {
```

Comment:
I don't think what you receive here is an object, it is a string value which shows the attribute name